tardis/tardis_portal/tests/test_collectstatic.py

```diff
@@ -10,14 +11,22 @@ class CollectstaticTest(TestCase):
 
     def setUp(self):
         self.STATIC_ROOT = mkdtemp()
+        self.NPM_ROOT_PATH = mkdtemp()
         self.cmd = collectstatic.Command()
 
+        # Required when using django-npm's 'npm.finders.NpmFinder'
+        # in settings.STATICFILES_FINDERS:
+        mkdir(path.join(self.NPM_ROOT_PATH, 'node_modules'))
+
     def tearDown(self):
         if path.exists(self.STATIC_ROOT):
             shutil.rmtree(self.STATIC_ROOT)
```

Comment:
Should we be cleaning up `self.NPM_ROOT_PATH` in the `tearDown` as well?

Reply:
Thanks @keithschzule, the self.NPM_ROOT_PATH clean up has been implemented in 3e237ef1b86374986d5b8e70dfba3df71b588b7a